cypress-tests/cypress/support/commands.js

```diff
@@ -1018,26 +1018,22 @@ Cypress.Commands.add(
 
       expect(response.headers["content-type"]).to.include("application/json");
       if (response.status === 200) {
-        expect(response.body.client_secret, "client_secret").to.equal(
-          response.body.client_secret
-        );
-        expect(response.body.payment_method_id, "payment_method_id").to.equal(
-          response.body.payment_method_id
-        );
-        expect(response.body.merchant_id, "merchant_id").to.equal(
-          response.body.merchant_id
-        );
-        expect(response.body.payment_method, "payment_method").to.equal(
-          response.body.payment_method
-        );
+        expect(response.body.client_secret, "client_secret").to.not.be.null;
+        expect(response.body.payment_method_id, "payment_method_id").to.not.be
+          .null;
+        expect(response.body.merchant_id, "merchant_id").to.not.be.null;
```

Comment:
`L1007`, insert `const merchant_id = globalState.get("merchantId");

```suggestion
        expect(response.body.client_secret, "client_secret").to.include(`${payment_id}_secret_`);
        expect(response.body.payment_method_id, "payment_method_id").to.not.be
          .null;
        expect(response.body.merchant_id, "merchant_id").to.equal(merchant_id);
```

or, for this: `expect(response.body.client_secret, "client_secret").to.not.be.null.and.to.include(`${payment_id}_secret_`);`, we can directly valiadate something like below:

```js
expect(response.body.client_secret, "client_secret").to.equal(createPaymentMethodBody.client_secret);
```